otter/convergence/composition.py

```diff
@@ -67,7 +67,7 @@ def json_to_LBConfigs(lbs_json):
         if lb.get('type') != 'RackConnectV3':
             lbd[lb['loadBalancerId']].append(CLBDescription(
                 lb_id=str(lb['loadBalancerId']), port=lb['port']))
-    return lbd
+    return dict(lbd)
```

Comment:
Awesome :) Feel free to do that in a different PR if it ends up being messy (e.g. something is relying on it being mutable)